src/hydrusvideodeduplicator/__main__.py

```diff
@@ -68,6 +70,8 @@ def main(
         api_key = HYDRUS_API_KEY
     if not api_url:
         api_url = HYDRUS_API_URL
+    if not file_service_key:
+        file_service_key = HYDRUS_LOCAL_FILE_SERVICE_KEY
```

Comment:
@appleappleapplenanner I went back and forth on including this myself, actually! I landed on not including it, because I think that it doesn't actually change the value of `file_service_key` regardless of whether `HYDRUS_LOCAL_FILE_SERVICE_KEY` is set or not thanks to using `HYDRUS_LOCAL_FILE_SERVICE_KEY` as the default value on line 33. But I might well be missing something in my understanding here, so happy to include it.

Reply:
Oh, you're right. You can remove this and also remove the two lines above it for the api_url for the same reason. 

I actually hate how I'm handling env variables and config and the more I use typer the most I really don't like it. But, I don't see a better option right now :(. There's a lot of random places where constants and configuration are stored in the program, which is just because of my lack of experience in designing CLI programs. I hate designing programs with user interfaces!

Reply:
@appleappleapplenanner Yo, as another backend engineer, I feel you 😂 Whenever I make a program, I'm like: "...an API is a user interface, right?"

Reply:
@appleappleapplenanner Wild story, I actually wound up keeping all three lines, because I realized that using the environment variable as the CLI option default lead to some ugly --help text, so instead I switched it's default to None (which also better telegraphs that it's not required, I think)